src/utils/number.ts

```diff
@@ -15,11 +20,24 @@ export const formatNumberShort = (num: number): string => {
   }
 
   let formattedValue = value.toFixed(1)
+
+  if (Number(formattedValue) >= 1000) {
+    const tempNumber: string[] = []
+    const formattedValueAsArray = [...formattedValue]
+    formattedValueAsArray.forEach((value, idx) => {
+      tempNumber.push(value)
+      if (idx === 0 || (formattedValue.length - idx > 5 && idx % 3 === 0)) {
+        tempNumber.push(' ')
+      }
+    })
+    formattedValue = tempNumber.join('')
+  }
```

Comment:
This looks a bit dark magic-y, what's happening here? When would `Number(formattedValue) >= 1000` happen given the divisions we do above?

Reply:
This function adds whitespace between between every thousand. This condition works above 999T.
https://www.notion.so/Date-time-number-formats-in-Atlas-c5e7c7aec24e4ddfa83104ca4c2e924e

Reply:
Is this any different than `value.toLocaleString('pl')`? 🤔 

Btw, we will also be able to handle only up to 9000T with this code, anything above will be over JS integer safe limits so we will need to work with some kind of big numbers

Reply:
Although we can do second part on some different occasion